back/config/environments/staging.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'active_support/core_ext/integer/time'
```

Comment:
I've just noticed we have this in `config/application.rb` so maybe I am wrong about this being the 'fix' for staging?